FrontEnd/stores/post/actions.js

```diff
@@ -0,0 +1,9 @@
+// stores/post/actions.js
+export function setPosts(newPosts) {
+    this.posts = newPosts;
+  }
+  
+  export function clearPosts() {
+    this.posts = [];
+  }
+  
```

Comment:
For options api syntax we should keep the code in same file, in case we have convention to not exceed a certain lines of code we can split the store based on functionality instead of split based on pinia properties.

Reply:
modified and updated the postStore as suggested,now the postStore is setup in one file and added compostion api syntax and also followed the naming convention as suggested. please review the latest commit for the changes.